Resources/Private/JavaScript/@types/interactjs.d.ts

```
@@ -0,0 +1,27 @@
+declare type InteractjsDragEvent = {
+  dx: number;
+  dy: number;
+  target: Element;
+}
+
+declare module "interactjs" {
+
+  declare type Chainable = {
+    resizable(config: {}): Chainable;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)